src/hoc/Layout/Layout.test.js

```diff
@@ -1,11 +1,11 @@
-// import React from 'react';
+import React from 'react';
 
-// import { configure, shallow } from 'enzyme';
-// import Adapter from 'enzyme-adapter-react-16';
-// import { Layout } from './Layout';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { Layout } from './Layout';
 
-// configure({adapter: new Adapter()});
+configure({adapter: new Adapter()});
 
-//     it('should render <Layout /> when receiving ingredients', () => {
+    it('should render <Layout /> when receiving ingredients', () => {
```

Comment:
There is nothing inside it. whole file doesn't means anything.
Test coverage for this file is 0